test/functional/feature_config_args.py

```diff
@@ -92,15 +92,18 @@ def run_test(self):
         with open(conf_file, 'w', encoding='utf8') as f:
             f.write("datadir=" + new_data_dir + "\n")
             f.write(conf_file_contents)
-        self.nodes[0].assert_start_raises_init_error(['-conf=' + conf_file], 'Error reading configuration file: specified data directory "' + new_data_dir + '" does not exist.')
+
+        # Temporarily disabled, because this test would access the user's home dir (~/.bitcoin)
+        #self.nodes[0].assert_start_raises_init_error(['-conf=' + conf_file], 'Error reading configuration file: specified data directory "' + new_data_dir + '" does not exist.')
 
         # Create the directory and ensure the config file now works
         os.mkdir(new_data_dir)
-        self.start_node(0, ['-conf='+conf_file, '-wallet=w1'])
-        self.stop_node(0)
-        assert os.path.exists(os.path.join(new_data_dir, self.chain, 'blocks'))
-        if self.is_wallet_compiled():
-            assert os.path.exists(os.path.join(new_data_dir, 'regtest', 'wallets', 'w1'))
+        # Temporarily disabled, because this test would access the user's home dir (~/.bitcoin)
+        #self.start_node(0, ['-conf='+conf_file, '-wallet=w1'])
+        #self.stop_node(0)
+        #assert os.path.exists(os.path.join(new_data_dir, self.chain, 'blocks'))
+        #if self.is_wallet_compiled():
+        #    assert os.path.exists(os.path.join(new_data_dir, 'regtest', 'wallets', 'w1'))
```

Comment:
13687:
```suggestion
        #    assert os.path.exists(os.path.join(new_data_dir, self.chain, 'wallets', 'w1'))
```

 _ideally_ this should be fixed in 14324 which should be backported _after_ 13687

Reply:
But is that really important?

Reply:
would be cleaner/easier to review this way imo